templates/NOTES.txt

```diff
@@ -7,11 +7,18 @@ Consul with Kubernetes available here:
 https://www.consul.io/docs/platform/k8s/index.html
 
 
-Your release is named {{ .Release.Name }}. To learn more about the release, try:
+Your release is named {{ .Release.Name }}. To learn more about the release,
```

Comment:
```suggestion
Your release is named {{ .Release.Name }}.
```

Reply:
Done in commit : 1195a6c413892e82a8968100272845e3b3aa2db9